react-form-controls/my-app/src/RegistrationFormUncontrolled.js

```diff
@@ -3,9 +3,8 @@ export default function RegistrationFormUncontrolled() {
     event.preventDefault();
     const formData = new FormData(event.target);
 
-    const state = Object.fromEntries(formData.entries());
-
-    console.log(state);
+    const formValues = Object.fromEntries(formData.entries());
```

Comment:
Please look at step 5 again:  Instead of a variable state you will destructure const {username, password}  and then log those values.

`const {username, password} = Object.fromEntries(formData.entries());`